tests/unit/test_adapter.py

```diff
@@ -182,17 +178,17 @@ def test_acquire_connection(self, connection_cls):
         connection_cls.assert_called_once()
 
     @mock.patch("dbt.adapters.athena.connections.AthenaConnection")
-    def test_acquire_connection_exc(self, connection_cls, caplog):
-        caplog.set_level(logging.ERROR)
+    def test_acquire_connection_exc(self, connection_cls, dbt_error_caplog):
         connection_cls.side_effect = lambda **_: (_ for _ in ()).throw(Exception("foobar"))
         connection = self.adapter.acquire_connection("dummy")
         conn_res = None
-        with pytest.raises(FailedToConnectException) as exc:
+        with pytest.raises(ConnectionError) as exc:
             conn_res = connection.handle
+
         assert conn_res is None
         assert connection.state == ConnectionState.FAIL
-        assert exc.value.msg == "foobar"
-        assert "Got an error when attempting to open a Athena connection due to foobar" in caplog.text
+        assert exc.value.__str__() == "foobar"
```

Comment:
`msg` property is not available anymore ?

Reply:
Nope, but I found out that what we want is in the value.__str__